src/client/screens/PublishedMapsListScreen.tsx

```diff
@@ -0,0 +1,110 @@
+/** @jsx jsx */
+import { useEffect } from "react";
+import { connect } from "react-redux";
+import { Flex, jsx, Spinner, Box, Heading, Text } from "theme-ui";
+import { IProject, ProjectNest } from "../../shared/entities";
+import "../App.css";
+import { State } from "../reducers";
+import store from "../store";
+import SiteHeader from "../components/SiteHeader";
+import { globalProjectsFetch, globalProjectsFetchPage } from "../actions/projects";
+import { UserState } from "../reducers/user";
+import FeaturedProjectCard from "../components/FeaturedProjectCard";
+import PaginationFooter from "../components/PaginationFooter";
+
+interface StateProps {
+  readonly globalProjects?: readonly IProject[];
+  readonly pagination: {
+    readonly currentPage: number;
+    readonly limit: number;
+    readonly totalItems?: number;
+    readonly totalPages?: number;
+  };
+  readonly user: UserState;
+}
+
+const style = {
+  projects: {
+    pt: 4,
+    pb: 8,
+    flexDirection: "row",
+    width: "large",
+    mx: "auto",
+    "> *": {
+      mx: 5
+    },
+    "> *:last-of-type": {
+      mr: 0
+    },
+    "> *:first-of-type": {
+      ml: 0
+    }
+  },
+  featuredProjectContainer: {
+    display: "grid",
+    gridTemplateColumns: "repeat(4, 1fr)",
+    gridGap: "15px",
+    justifyContent: "space-between",
+    marginTop: "4"
+  },
+  pagination: {
+    cursor: "pointer"
+  }
+};
+
+const PublishedMapsListScreen = ({ globalProjects, user, pagination }: StateProps) => {
+  useEffect(() => {
+    !globalProjects && store.dispatch(globalProjectsFetch());
+  }, [globalProjects]);
+
+  return (
+    <Flex sx={{ height: "100%", flexDirection: "column" }}>
+      <SiteHeader user={user} />
+      <Box>
+        {globalProjects ? (
+          // @ts-ignore
+          <Box sx={style.projects}>
```

Comment:
Using `as const` for `style` will fix the typing issues.

```suggestion
} as const;

const PublishedMapsListScreen = ({ globalProjects, user, pagination }: StateProps) => {
  useEffect(() => {
    !globalProjects && store.dispatch(globalProjectsFetch());
  }, [globalProjects]);

  return (
    <Flex sx={{ height: "100%", flexDirection: "column" }}>
      <SiteHeader user={user} />
      <Box>
        {globalProjects ? (
          <Box sx={style.projects}>
```